src/DependencyInjection/HttplugExtension.php

```diff
@@ -112,6 +112,12 @@ public function load(array $configs, ContainerBuilder $container): void
             $container->removeAlias(ClientInterface::class);
         }
 
+        if (!$config['default_client']) {
+            $container->removeAlias('httplug.client');
+            $container->removeAlias('httplug.client.default');
+            $container->removeDefinition('httplug.client.default');
```

Comment:
i don't think we should remove this *definition*. it has no semantic meaning for symfony, and if i define a client that i call `default` it would be very unexpected to just nuke my definition.

however, we should additionally remove the httplug.psr18_client.default alias (from the main_classes)

Reply:
That's a good catch indeed. It's naive for me to just remove this, because I assume it's the default one created by the bundle.